.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test LocalGov Drupal
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone drupal_container
+        uses: actions/checkout@v2
+        with:
+          repository: localgovdrupal/drupal-container
+          ref: master
+
+      - name: Create LocalGov Drupal project
+        run: composer create-project --stability dev localgovdrupal/localgov-project html
+
+      - name: Checkout ${{ github.event.repository.name }} branch to test
```

Comment:
Looks like ${{ github.event.repository.name }} here is referring to the Git repository name rather than the Git branch name.

Reply:
Hi @Adnan-cds 

As I understand it, that line just prints out what is about to happen, checking out the repo that we are testing.

The next line 

```uses: actions/checkout@v2```

checks out the commit that triggers the action, which I think in this case is the merge commit against which to run the tests.

Does that make sense? Have I understood this right?

Reply:
https://github.com/actions/checkout/blob/main/README.md suggests that the ```ref: ''``` option defaults to the SHA of the current event if not specified.

Reply:
Hi Finn,
If the ${{ github.event.repository.name }} mentioned in line 24 can print the Git **branch name** then all is fine.  I am saying this because rest of line 24 says "...branch to test".